sway/input/seat.c

```diff
@@ -725,14 +726,25 @@ static void seat_apply_input_config(struct sway_seat *seat,
 		ic == NULL ? MAPPED_TO_DEFAULT : ic->mapped_to;
 
 	switch (mapped_to) {
-	case MAPPED_TO_DEFAULT:
+	case MAPPED_TO_DEFAULT:;
 		/*
 		 * If the wlroots backend provides an output name, use that.
 		 *
 		 * Otherwise, try to map built-in touch and tablet tool devices to the
 		 * built-in output.
 		 */
-		mapped_to_output = sway_device->input_device->wlr_device->output_name;
+		struct wlr_input_device *dev = sway_device->input_device->wlr_device;
+		switch (dev->type) {
+		case WLR_INPUT_DEVICE_POINTER:
+			mapped_to_output = dev->pointer->output_name;
+			break;
+		case WLR_INPUT_DEVICE_TOUCH:
+			mapped_to_output = dev->touch->output_name;
+			break;
+		default:
```

Comment:
This is missing a case for tablet tool devices.

Reply:
Ah, I guess we don't actually have an `output_name` field for these yet?

Reply:
Nope, only tablet and pointer.